users/views.py

```diff
@@ -0,0 +1,101 @@
+import json
+import re
+import jwt
+import bcrypt
+
+
+from django.http      import JsonResponse
+from django.views     import View
+from django.db.models import Q
+
+from .models          import User
+from my_settings      import SECRET_KEY, ALGORITHM
+
+class SignupView(View):
+    def post(self, request):
+        vali_nickname     = '^[a-zA-Z0-9가-힣]{3,}$'
+        vali_email        = '^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
+        vali_password     = '^(?=.*[A-Za-z])(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!%*#?&]{6,}$'
+        validate_nickname = re.compile(vali_nickname)
+        validate_email    = re.compile(vali_email)
+        validate_password = re.compile(vali_password)
+        
+
+        try:
+            data       = json.loads(request.body)
+            password   = data.get('password')
+            password1  = data.get('password1')
+            encode_pw  = password.encode("utf-8")
+            encrypt_pw = bcrypt.hashpw(encode_pw, bcrypt.gensalt()).decode("utf-8")
+
+            if not validate_nickname.match(data['nickname']):
+                return JsonResponse(
+                    {"MESSAGE" : "Invalid_nickname"}, status = 400
```

Comment:
```suggestion
                    {"message" : "INVALID_NICKNAME"}, status = 400
```
여기서 메세지는 에러 코드 메세지 같은것이라. 소문자보다는 대문자로 통일해주시구요. 항상 규칙성 있게, 작성 부탁드립니다. 
사실 현업에서는 특별한 상황이 아니라면, status 코드면 충분합니다. 
또한 키는 소문자로 해주시구요!
response 메세지는 이 API 엔드포인트를 이용하는 client인 프론트엔드 서버 개발자를 위한 메세지이지, 고객에게 전달되는게 아니므로. 이 조차도 규격을 가진 코드로 통일시켜야 한다는걸 명심해주세요.

Reply:
아래에 해당하는 메세지들을 변경해주세요.